assets/css/project.css

```diff
@@ -35,7 +35,17 @@ html[data-theme = 'dark']
 body{
```

Comment:
what is the `.swp` file for?

Reply:
I'm checking.
I don't know how it was added from my side

Reply:
@smaranjitghose its done.